test/test_win32_service.rb

```diff
@@ -10,14 +10,18 @@
 
 class TC_Win32_Service < Test::Unit::TestCase
   def self.startup
+    @@win_ver = Win32::Service.windows_version
     @@host = Socket.gethostname
-    @@service_name = 'stisvc'
-    @@elevated = Win32::Security.elevated_security?
+    @@service_name = @@win_ver < 6 ? 'Schedule' : 'LanmanServer'
+    # win32-security 0.3.1 crashes on 2003, so just assume elevated there
+    @@elevated = false
```

Comment:
Note that the `win32-security` gem 0.3.1 has issues calling `elevated_security?` on Windows 2003, so the tests here opt to set `@@elevated = false` by default to workaround that behavior.  Ideally, `win32-security` would be changed to address this.

In Puppet, we use a slightly more involved mechanism for checking elevated:
https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/user.rb#L10-L52
https://github.com/puppetlabs/puppet/blob/master/lib/puppet/util/windows/process.rb#L183-L205